ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Common/tasks/main.yml

```diff
@@ -13,10 +13,15 @@
     path: C:\openjdk
     state: directory
   tags: basic_config
-  
+
 - name: Create cmd.exe shortcut
   win_shortcut:
     src: C:\Windows\System32\cmd.exe
     dest: C:\Users\Public\Desktop\cmd.lnk
     icon: C:\Windows\System32\cmd.exe,0
   tags: basic_config
+
+- name: Set Hostname
+  raw: netdom renamecomputer "$env:COMPUTERNAME" /Newname "{{ inventory_hostname }}" -force
+  when: Domain == "adoptopenjdk.net"
```

Comment:
Would this step not be wanted if the domain wasn't adoptopenjdk.net?  Example, a developer running this against their own cloud instance.

Reply:
I have just followed what @bblondin did on the unix playbook

Reply:
@bblondin change has a case for Domain == "adoptopenjdk.net" and Domain != "adoptopenjdk.net"